docs/operator-guides/upgrading-airbyte.md

```diff
@@ -20,19 +20,31 @@ If you inadvertently upgrade to a version of Airbyte that is not compatible with
    docker-compose down
    ```
 
-2. Turn back on the Airbyte web app, server, and db \(using the "old" working version from which you are upgrading from\).
+2. Upgrade the docker instance to new version.
+
+   i. If you are running Airbyte from a cloned version of the Airbyte GitHub repo and want to use the current most recent stable version, just `git pull`.
+
+   ii. If you are running Airbyte from downloaded `docker-compose.yaml` and `.env` files without a GitHub repo, run `wget -N https://raw.githubusercontent.com/airbytehq/airbyte/master/{.env,docker-compose.yaml}` to pull the latest versions and overwrite both files.
+
+7. Bring Airbyte back online.
 
    ```bash
-   docker-compose up -d db server webapp
+   docker-compose up
    ```
 
-   _Note: the `-d` flag in the previous command is optional and is meant to run the docker services in the background. This would let you free to keep interacting with the terminal and type the next commands in the tutorial._
+8. Complete Preferences section. In the subsequent setup page click "Skip Onboarding". Navigate to the Admin page in the UI. Then go to the Configuration Tab. Click Import. This will prompt you to upload the migrated archive to Airbyte. After this completes, your upgraded Airbyte instance will now be running with all of your original configuration.
+
+This step will throw an exception if the data you are trying to upload does not match the version of Airbyte that is running.
 
-3. Switching over to your browser, navigate to the Admin page in the UI. Then go to the Configuration Tab. Click Export. This will download a compressed back-up archive \(gzipped tarball\) of all of your Airbyte configuration data and sync history locally.
+## Upgrading \(K8s\)
+
+This process is similar to the Docker upgrade process with several changes to account for the Kubernetes resources.
```

Comment:
Shouldn't this say that Automatic Migration is not supported for Kubernetes?

A user has to follow the following steps to get this working.

Reply:
done